Source/UI/NewUI/PaymentForm/View/SecurityCodeView.swift

```diff
@@ -0,0 +1,75 @@
+//
+//  SecurityCodeView.swift
+//  
+//
+//  Created by Ehab Alsharkawy
+//  Copyright © 2022 Checkout. All rights reserved.
+
+import UIKit
+import Checkout
+
+public final class SecurityCodeView: UIView {
+
+  private let securityCodeCount = 4
```

Comment:
Is this value supposed to be hardcoded and constant? What happens when a card has 3 CVV characters?

Reply:
Good question : I think we should rename this to maxSecurityCodeCount as its actually a max value it can also be 3, see the check on line 53 below.